packages/fluent-theme/src/components/sendbox/index.module.css

```diff
@@ -3,20 +3,31 @@
   font-family: var(--webchat-fontFamilyBase);
   padding: 0 10px 10px;
   text-rendering: optimizeLegibility;
+
+  --webchat-sendbox-attachment-area-active: ;
 }
 
 :global(.webchat-fluent) .sendbox__sendbox {
   background-color: var(--webchat-colorNeutralBackground1);
   border-radius: var(--webchat-borderRadiusLarge);
   border: 1px solid var(--webchat-colorNeutralStroke1);
   display: grid;
+  gap: 6px;
+  grid-template:
+    [telephone-keypad-start] 'text-area'
+    var(--webchat-sendbox-attachment-area-active) [telephone-keypad-end]
+    'controls' / [telephone-keypad] 1fr
+  ;
   font-family: var(--webchat-fontFamilyBase);
   font-size: 14px;
-  gap: 6px;
   line-height: 20px;
   padding: 8px;
   position: relative;
 
+  &:has(.sendbox__attachment--in-grid) {
```

Comment:
Instead of `in-grid`, should we call it `visible`, `shown`, `hidden` (the other way) instead?

Reply:
We want to know that not only the element is visible, but the corresponding area is in use